pom.xml

```diff
@@ -149,7 +149,7 @@
     <httpcomponents.client.version>4.5.6</httpcomponents.client.version>
     <httpcomponents.core.version>4.4.10</httpcomponents.core.version>
     <ivy.version>2.4.0</ivy.version>
-    <jackson.version>2.10.0</jackson.version>
+    <jackson.version>2.10.3</jackson.version>
```

Comment:
Small nit - not sure why 2.10.3 was chosen - but 2.10.5 is out - this would address any issues w/ snakeyaml <1.26 which is pulled in by `jackson-dataformat-yaml` - this was fixed in 2.10.4+.